web/packages/shared/components/UnifiedResources/CardsView/LoadingCard.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import { Box, Flex } from 'design';
+import { ShimmerBox } from 'design/ShimmerBox';
+
+import { LoadingItemProps } from '../types';
+
+export function LoadingCard({ delay = 'none' }: LoadingItemProps) {
+  const [canDisplay, setCanDisplay] = useState(false);
+
+  useEffect(() => {
+    const displayTimeout = setTimeout(() => {
+      setCanDisplay(true);
+    }, LoadingItemDelayValueMap[delay]);
+    return () => {
+      clearTimeout(displayTimeout);
+    };
+  }, []);
+
+  if (!canDisplay) {
+    return null;
+  }
+
```

Comment:
I think we don't need the timeout here anymore once @gzdunek changes go through as he updated the way the card loader works. 

ListViewSkeleton looks great

premptive approve once the other PR goes in and conflicts resolved here